packages/fx-core/src/component/generator/spfx/utils/questions.ts

```diff
@@ -117,3 +121,36 @@ export const spfxPackageSelectQuestion: Question = {
     },
   },
 };
+
+export const spfxSolutionQuestion: Question = {
+  type: "singleSelect",
+  name: SPFXQuestionNames.spfx_solution,
+  title: getLocalizedString("plugins.spfx.questions.spfxSolution.title"),
+  staticOptions: [
+    { id: "new", label: getLocalizedString("plugins.spfx.questions.spfxSolution.createNew") },
+    {
+      id: "import",
+      label: getLocalizedString("plugins.spfx.questions.spfxSolution.importExisting"),
+    },
+  ],
+};
+
+export function spfxImportFolderQuestion(): Question {
+  return {
+    type: "folder",
+    name: SPFXQuestionNames.spfx_import_folder,
+    title: getLocalizedString("core.spfxFolder.title"),
+    placeholder: getLocalizedString("core.spfxFolder.placeholder"),
+  };
+}
+
+export const skipAppName: Question = {
+  type: "func",
+  name: SPFXQuestionNames.skip_app_name,
+  title: "Set app name to skip",
```

Comment:
Does this one need to be localized?

Reply:
This is an invisible question so no need to localize